README.md

```diff
@@ -3,9 +3,20 @@
 `mktrayicon` is a simple proxy program that lets you create and modify system
 tray icons without having to deal with a graphical toolkit like GTK.
 
-`mktrayicon` expects to be given a name pipe (FIFO) file path when it is
-started, and you control your status icon by writing to this named pipe. *Note
-that the pipe should already be created before you call `mktrayicon`*.
+`mktrayicon` can be used two ways: To create an icon that is controlled by a named pipe
+or, more simply, to create a non-interactive icon using the specified tooltip string (which can be an empty string if you don't need a tooltip). 
```

Comment:
Does it explicitly have to be the empty string (i.e., `-t ""`), or can it be simply omitting the `-t`?

Reply:
-t is what signals that the icon is non-interactive. Both `mktrayicon -i <icon> -t ""` and `mktrayicon -i <icon> -t` work.